core/src/index/thirdparty/faiss/gpu/impl/IVFFlat.cu

```diff
@@ -102,6 +106,16 @@ IVFFlat::copyCodeVectorsFromCpu(const float* vecs,
     if (stream != 0) {
         streamWait({stream}, {0});
     }
+    std::chrono::high_resolution_clock::time_point time5 = std::chrono::high_resolution_clock::now();
+
+    double span1 = (std::chrono::duration<double, std::micro>(time2 - time1)).count();
+    double span2 = (std::chrono::duration<double, std::micro>(time3 - time2)).count();
+    double span3 = (std::chrono::duration<double, std::micro>(time4 - time3)).count();
+    double span4 = (std::chrono::duration<double, std::micro>(time5 - time4)).count();
+    std::cout << "Span1: " << span1 * 0.001 << "ms" << std::endl;
+    std::cout << "Span2: " << span2 * 0.001 << "ms" << std::endl;
+    std::cout << "Span3: " << span3 * 0.001 << "ms" << std::endl;
+    std::cout << "Span4: " << span4 * 0.001 << "ms" << std::endl;
```

Comment:
Each search the time information will be printed out on stdout?

Reply:
What we need is millisecond.